MagickCore/cache.c

```diff
@@ -3836,6 +3836,8 @@ MagickExport MagickBooleanType PersistPixelCache(Image *image,
     Clone persistent pixel cache.
   */
   clone_info=(CacheInfo *) ClonePixelCache(cache_info);
+  if (clone_info == (CacheInfo *) NULL)
```

Comment:
We don't need this check. `ClonePixelCache` calls `AcquirePixelCache` that will throw a fatal exception when the pixel cache could not be acquired.

Reply:
You are right.